fix(curriculum): add missing tests for step 136 of JS RPG game (#52793)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8f1d5f5ddbf74c07f733b.md

# --hints--

> update method should be called

Reviewer: ```suggestion
You should call the `update` function inside the `lose` function.
```

```js

> assert.match(lose.toString(), /update/)

Reviewer: So technically this test only checks if the camper wrote `update` inside the `lose` function. 
It doesn't test if the camper calls it though.
We should update this test to check if the `update` function called

This should work

```suggestion
assert.match(lose.toString(), /update\(/);
```

```

> locations[5] should be passed as argument to update method

Reviewer: ```suggestion
Your `update` function should have `locations[5]` for the argument.
```

```js

> assert.match(lose.toString(), /update\(locations\[5\]\)/)

Reviewer: add in missing semicolon 

```suggestion
assert.match(lose.toString(), /update\(locations\[5\]\)/);
```
